.github/workflows/build-binaries.yaml

```diff
@@ -31,11 +31,13 @@ jobs:
             target: x86_64-unknown-linux-musl
             output: libyggdrasilffi.so
             name: libyggdrasilffi_x86_64-musl.so
+            compiler: x86_64-linux-musl-gcc
             cross: true
           - os: ubuntu-latest
             target: aarch64-unknown-linux-musl
             output: libyggdrasilffi.so
             name: libyggdrasilffi_arm64-musl.so
+            compiler: aarch64-linux-musl-gcc
```

Comment:
I thought it was to not rely on libgcc? your compiler choice sounds like it does depend on it :)